src/assets/server/rest/v1/schemas/transaction/transactions-get.json

```diff
@@ -0,0 +1,87 @@
+{
+  "title": "Get Transactions Request",
+  "type": "object",
+  "properties": {
+    "ChargeBoxID": {
+      "$ref": "chargingstation.json#/definitions/id"
+    },
+    "Issuer": {
+      "type": "boolean",
+      "sanitize": "mongo"
+    },
+    "ConnectorID": {
+      "$ref": "chargingstation.json#/definitions/connector/properties/connectorId"
+    },
+    "SiteAreaID": {
+      "$ref": "common.json#/definitions/ids"
+    },
+    "SiteID": {
+      "$ref": "common.json#/definitions/ids"
+    },
+    "UserID": {
+      "$ref": "common.json#/definitions/ids"
+    },
+    "TagID": {
+      "$ref": "tag.json#/definitions/id"
```

Comment:
Same thought like for the chargebox ids. Do we need a differentiation between singular and plural?
It works currently because the string can be split before the database request even if the definition is singular.

Reply:
Added a specific parameters "ids" and enforced the "id" parameter